changelog.txt

```diff
@@ -1,5 +1,8 @@
 V.Next
 ---------
+
+Version 17.5.0
+---------
 - [MINOR] Update logic for matching requested claims for AT (#2401)
 - [MINOR] Updating YubiKit and CredMan versions (#2417)
 - [PATCH] Adding check for OS version for passkeys (#2419)
```

Comment:
I merged #2419 and #2426 after the branches were cut, so they should be under V.next